ext/opcache/Optimizer/pass1_5.c

```diff
@@ -409,6 +409,11 @@ void zend_optimizer_pass1(zend_op_array *op_array, zend_optimizer_ctx *ctx)
 				    send2_opline &&
 				    Z_TYPE(ZEND_OP1_LITERAL(send2_opline)) <= IS_STRING) {
 
+					if (collect_constants
```

Comment:
The check inside the if looks to have been used more than once. Could this possible be pulled into its own (possible static?) function? It also might help with understanding exactly what this check is doing.